solutions/src/main/java/ru/sberbank/school/task13/cacheproxy/exception/CacheProxyIOException.java

```diff
@@ -0,0 +1,7 @@
+package ru.sberbank.school.task13.cacheproxy.exception;
+
+public class CacheProxyIOException extends CacheProxyException {
```

Comment:
Abbreviation in name &#39;CacheProxyIOException&#39; must contain no more than &#39;2&#39; consecutive capital letters.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a120724309>*